vale/dictionary.jsonnet

```
@@ -227,6 +227,7 @@ local newWord(word, affixes, po) = {
     newWord('UTF', '', 'noun') { acronym: true, established_acronym: true },
     newWord('Velero', '', 'noun') { description: 'Velero is an open source tool to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes.', product: true },
     newWord('Vite', '', 'noun') { description: 'Next Generation Frontend Tooling (https://vitejs.dev/)', product: true },
+    newWord('viewport', '', 'noun'),
```

I'm guessing there is a plural form which is what the 'S' affix does. I'm planning to write https://github.com/grafana/writers-toolkit/issues/669 this week to help with this.

```suggestion
    newWord('viewport', 'S', 'noun') { description: 'A viewport is a polygon viewing region in computer graphics (https://en.wikipedia.org/wiki/Viewport)' },
```

---

I was't really sure with plurals... I'll watch for this in future reviews.